src/meander/syntax/alpha.clj

```diff
@@ -141,7 +141,7 @@
 (defn zero-or-more-symbol?
   [x]
   (and (simple-symbol? x)
-       (re-matches #"\.\.+" (name x))))
+       (re-matches #"\.\.\." (name x))))
```

Comment:
This change wasn't strictly necessary. But as mentioned in the commit message I want to handle use of the one-or-more operator improperly `(1 .. ?x)`. The user could have meant either zero-or-more and forgot a dot or one-or-more and forgot the number. In that change as well as this I made the regex a little different to accommodate the error messages. After this PR is merged, I can make one for one-or-more and we can talk about the one ugly conditional I had to add.

Reply:
I made a small tweak to this such that we expect three or more dots.